domain/src/main/java/org/oppia/android/domain/exploration/ExplorationProgressController.kt

```diff
@@ -1106,6 +1126,9 @@ class ExplorationProgressController @Inject constructor(
     fun endState() {
       stateAnalyticsLogger?.logEndCard()
       explorationAnalyticsLogger.endCard()
+      isContinueButtonAnimationSeen?.let {
```

Comment:
So I'd expect this to be a var, actually, since it needs to be set to false at endState to avoid showing the animation after the first state.

Reply:
done